.github/workflows/ci.yml

```diff
@@ -48,11 +48,12 @@ jobs:
           override: false
       - name: Add target
         run: rustup target add ${{ matrix.target }}
-      - name: cargo build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --no-default-features --features batch,dev-graph,gadget-traces --target ${{ matrix.target }}
+      - name: Make the script files executable
+        run: |
+          chmod +x .github/scripts/wasm-target-test-build.sh
```

Comment:
This should not be needed.  Can you set `+x` on the file itself and push it?  I think the file executable flag will be propagated into git.

Reply:
Done https://github.com/privacy-scaling-explorations/halo2/pull/298/commits/e7da4d66de0fe9f606c6f1ab193253ee2b016610